package.json

```diff
@@ -24,7 +24,7 @@
     }
   },
   "files": [
-    "README.md",
+    "README.adoc",
     "LICENSE",
```

Comment:
These are implicit. See https://docs.npmjs.com/files/package.json#files:
> Certain files are always included, regardless of settings:
> - package.json
> - README
> - CHANGES / CHANGELOG / HISTORY
> - LICENSE / LICENCE
> - NOTICE
> - The file in the “main” field

Reply:
I suspected this, but didn't check. Thanks!